src/programs/federal/snap.js

```diff
@@ -49,140 +51,170 @@ const getSNAPBenefits = function (client, timeframe) {
 }; // End getSNAPBenefits()
 
 
-// ======================
-// ESTABLISH OBJECT FOR BENEFIT HELPERS
+/** Abstractions for main function calculations.
+ *     Many aren't required, but make unit testing
+ *     easier by taking away the need for unit tests
+ *     to mess around with actual client values.
+ * 
+ * @namespace
+ *
+ * @note Almost all require the `current` or `future`
+ *     prop of the typical `client` object as an
+ *     argument.
+ */
 var SNAPhelpers = {},
     hlp         = SNAPhelpers;
```

Comment:
While we're making this file clearer: could we move these declarations up above `getSNAPBenefits`? I was scratching my head trying to figure out where `hlp` was coming from.

Reply:
Good idea!